app/controllers/static_pages_controller.rb

```diff
@@ -23,7 +23,7 @@ def set_preview_events
   end
 
   def upcoming_events
-    Event.order(:start_time).limit(5)
+    Event.where(start_time: Time.now..(Time.now + 1.month)).limit(3).order(start_time: :asc)
```

Comment:
Can we show only the events from the week, and if there is not an event for the day, show that there is none. Example:
```
Monday: pic / summary
Tuesday: No Zagaku Scheduled
Wednesday: pic / summary
Thursday: pic / summary
```